Assets/Scripts/PlayerController/PlayerStats.cs

```diff
@@ -0,0 +1,86 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerStats : MonoBehaviour
```

Comment:
Same goes here, no need for this dictoinary of strings. also maybe write public float one time, just split it in few rows.